wagtail/contrib/search_promotions/templates/wagtailsearchpromotions/includes/searchpromotion_form.html

```diff
@@ -10,7 +10,7 @@
         <button type="button" class="button button--icon text-replace" id="{{ form.DELETE.id_for_label }}-button">{% icon name="bin" %}{% trans "Delete" %}</button>
     {% endfragment %}
 
-    {% trans "Recommended page" as heading %}
+    {% trans "Search result" as heading %}
```

Comment:
```suggestion
    {% trans "Recommended search result" as heading %}
```

A reminder that this feature is about Search promotions (aka recommended search results), so changing this to simply 'Search result' is not really correct.